skills/csharp/emailskill/Responses/ShowEmail/ShowEmailTexts.it-it.lg

```diff
@@ -13,3 +13,6 @@
 # ReadOutMessage.Speak(emailDetailsWithContent)
 - Ecco i dettagli dell'e-mail: ${emailDetailsWithoutContent}
 
+# ActionPrompt.Text
+- To take an action on an email, first select it, then you can delete, forward, or reply.
```

Comment:
can you use Google translate to translate these to respective languages?

Reply:
sure. fixed